app/assets/stylesheets/pghkids.scss

```diff
@@ -0,0 +1,40 @@
+@import url('http://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800,600');
+// from bower_components
+@import "breakpoint-sass/stylesheets/breakpoint";
+
+@import "library/reset";
+@import "library/grid";
+// @import "library/variables";
```

Comment:
I do not feel strongly either way. Eventually the other sites will be moved into the skins folder and everything will be the same. I feel that it's nicer to have the variables in the skins file but it's not a must-have.